qml/Ping360Visualizer.qml

```diff
@@ -379,9 +379,22 @@ Item {
                 onCurrentTextChanged: waterfall.theme = currentText
             }
 
+            CheckBox {
+                id: headingCorrection
```

Comment:
nitpick: it is funny to have two terms for the same thing: heading correction and heading integration

Reply:
Done.